src/libs/actions/Modal.js

```diff
@@ -1,6 +1,22 @@
 import Onyx from 'react-native-onyx';
 import ONYXKEYS from '../../ONYXKEYS';
 
+let closeModal;
+
+/**
+ * Allows other parts of the app to call modal close function
+ *
+ * @param {Function} [onClose]
+ */
+function setModalClose(onClose) {
```

Comment:
I feel like this function needs a better name. If not, a better description. If we leave it like this, it should be call `setCloseModal`, which it's the name of the parameters